server/mongo.js

```diff
@@ -3,7 +3,7 @@ const env = require('./env/environment');
 
 mongoose.Promise = global.Promise;
 
-const mongoUri = `mongodb://${env.dbName}.documents.azure.com:${env.cosmosPort}/?ssl/=true`;
+const mongoUri = `${env.dbName}:${env.key}@${env.dbName}.mongo.cosmos.azure.com:${env.cosmosPort}/?ssl=true`;
```

Comment:
@Mahyar1982 it should have "mongodb://" per the portal connection string. 

![image](https://user-images.githubusercontent.com/14284405/101214608-252dfb80-3631-11eb-9434-a4da3ec612ea.png)


```suggestion
const mongoUri = `mongodb://${env.dbName}:${env.key}@${env.dbName}.mongo.cosmos.azure.com:${env.cosmosPort}/?ssl=true`;
```

Reply:
@SnehaGunda thank you for the tip. It is committed.